packages/domain/src/use-cases/profile/UpdateProfileDetails.ts

```diff
@@ -1,18 +1,10 @@
-import { ProfileId, TransactionKind } from '../../entities';
+import { TransactionKind } from '../../entities';
 import { DelegableSigning } from '../transactions/DelegableSigning';
 
-export type ProfileAttributeValue = boolean | Date | string | number;
-
-export type PartialAttributesUpdate = Record<string, ProfileAttributeValue | null>;
-
 export type UpdateProfileDetailsRequest = {
-  attributes?: PartialAttributesUpdate;
-  bio?: string | null;
-  coverPicture?: string | null;
-  delegate: boolean;
+  metadataURI: string;
   kind: TransactionKind.UPDATE_PROFILE_DETAILS;
-  name: string;
-  profileId: ProfileId;
+  delegate: boolean;
 };
 
 export class UpdateProfileDetails extends DelegableSigning<UpdateProfileDetailsRequest> {}
```

Comment:
Should we rename this `SetProfileMetadata`?

Reply:
sure